docs/self-managed/identity/troubleshooting/common-problems.md

```diff
@@ -2,11 +2,12 @@
 id: common-problems
 title: "Common problems"
 sidebar_label: "Common problems"
+description: "Learn about solutions if Identity is unable to connect to Keycloak, or having issues getting Keycloak started."
```

Comment:
Here the descriptions are pretty specific, could we generalise this so the description remains fully relevant to the future content? Something like:
```
Learn about solutions to common issues in Identity
```

Reply:
Resolved in latest commit 👍